manifest.json

```diff
@@ -11,16 +30,13 @@
 
     "browser_action": {
         "default_icon": {
-            "16": "images/icon99.png",
-            "24": "images/icon99.png",
-            "32": "images/icon99.png"
+            "16": "images/icon16.png",
+            "24": "images/icon24.png",
+            "32": "images/icon32.png"
         },
         "default_title": "gh05t3ry",
         "default_popup": "popup.html"
-    },
-
-    "background": {
-        "js": ["background.js"]
     }
 
+
```

Comment:
You can cut these blanks - you only need a newline after the final closing curly brace below.